src/components/Modal/Modal.tsx

```diff
@@ -79,17 +79,24 @@ type ModalContentProps = {
   onClose: () => void;
   // Design API
   /**
-   * Max size of the modal, which responds to the viewport
-   *
-   * **Default is `"lg"`**.
+   * Determine how the height of the modal container is calculated:
+   * - `"fixed"` applies the fixed size, which will not adjust
```

Comment:
Would be good to refer to the size property if that's where fixed size comes from.

Reply:
fair point. I think a better word to use here is s/size/dimensions, since each `size` value has different dimensions as well. Will reword a bit